NARFO_BE/NARFO_BE/Models/SalesRepsPrototype.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NARFO_BE.Models
+{
+    public partial class SalesRepsPrototype
+    {
+        public string Firstname { get; set; }
+        public string SURNAME { get; set; }
+
+        public SalesRepsPrototype(string Firstname, string SURNAME) { this.Firstname = Firstname; this.SURNAME = SURNAME; }
```

Comment:
I prefer initiating the variables in the constructor on the next line, but again its preference